src/perform_search.py

```diff
@@ -5,14 +5,14 @@
 from .map_refseq_ids import map_refseq_ids_to_kbase
 
 
-def perform_search(sketch_path, db_name):
+def perform_search(sketch_path, db_name, n_max_results):
     """
     Make a request against the AssemblyHomologyService to do a search with a generated sketch file.
     """
     print('starting search request...')
     start_time = time.time()
     homology_url = os.environ.get('KBASE_HOMOLOGY_URL', 'https://homology.kbase.us')
-    path = '/namespace/' + db_name + '/search'
+    path = '/namespace/' + db_name + '/search' + '?max=' + str(n_max_results) 
```

Comment:
@slebras Oh I missed something here. You can pass url query string params more cleanly using the `params` keyword argument in requests.post. I will just move this on master.